zyngui/zynthian_gui_mixer.py

```diff
@@ -726,6 +726,13 @@ def __init__(self):
 		self.main_canvas.tag_bind("reset_balance_button", "<ButtonRelease-1>", self.on_reset_balance_release)
 		self.main_canvas.tag_bind("cancel_button", "<ButtonPress-1>", self.on_cancel_press)
 
+		# Horizontal scroll (via mouse wheel) area
+		legend_height = self.visible_mixer_strips[0].legend_height 
+		self.horiz_scroll_bg = self.main_canvas.create_rectangle(0, self.height - self.legend_height, self.width, self.height, width=0)
```

Comment:
This blocks mouse events reaching the strips so blocks touch navigation to control view / layer options. Need to find a way to provide both event handlers.